### PR TITLE
Suggestions for "survey.md"

### PR DESCRIPTION
**Pull request by WuraolaOyewusi**

### CODE REVIEW

File: SURVEY.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
-As I mentioned earlier we are a diverse group and that is what the pandas contributors were lacking of . And here we are .There are many constrains for each of us to participate in the program fully. It would be great if we have a scholarship which will support us to be a full time pandas contributor (in my opinion, if you have anything different in mind please share :))
-So lets participate in a survey which will help understand us better. The main aim of the survey is to get a better understanding of ourselves like who we are, where are we from, what  we are doing , do we need a scholarship  ,if yes, why etc
+Happy to be one of the energetic fellows in this program. We are a diverse team who truly want to make a mark.Regardless of where we are and what we know, we have the same mission and will be there to support each other ***sharing is always caring :)***.
```
Comment (martinagvilas) [inline SURVEY.md]: 1) I believe the correct expression is `leave a mark`, not `make a mark`

2) There is a space missing between `mark.` and `Regardless`

File: SURVEY.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
+The purpose of this program is to improve diversity in pandas contributors.There are many constraints for full participation in the program and It would be great if there is a scholarship that can support full time contribution to pandas (in my opinion, if you have anything different in mind please share :))
```
Comment (martinagvilas) [inline SURVEY.md]: 1. There is a space missing between `contributors.` and `There`
2. `It` should not be capitalized
3. There is an additional space between `(` and `in my opinion`

File: SURVEY.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
...
+So let's participate in a survey which will help understand us better. The main aim of the survey is to get a better understanding of ourselves like who we are?,where are we from?,what we are doing?,do we need a scholarship?,if yes,why etc
```
Comment (martinagvilas) [inline SURVEY.md]: 1. I would add a `:` after `like`
2. I would add a space between each comma and the next question

**Review by martinagvilas — CHANGES_REQUESTED** [review]
I think this is a good improvement of the text @WuraolaOyewusi, I just made some minor suggestions.

File: SURVEY.md (hunk 1 above)
Comment (WuraolaOyewusi) [reply SURVEY.md]: Ok @martinagvilas . I will fix the suggested changes. I was trying to keep as  much of the original text as possible.

File: SURVEY.md (hunk 3 above)
Comment (WuraolaOyewusi) [reply SURVEY.md]: done

File: SURVEY.md (hunk 2 above)
Comment (WuraolaOyewusi) [reply SURVEY.md]: done

File: SURVEY.md (hunk 1 above)
Comment (martinagvilas) [reply SURVEY.md]: makes sense!

**Review by martinagvilas — APPROVED** [review]

**Pull request merged**